package.json

```diff
@@ -1,102 +1,107 @@
-    "ember-composable-helpers": "^2.0.3",
-    "ember-concurrency": "^0.8.12",
-    "ember-data": "~2.18.0",
-    "ember-export-application-global": "^2.0.0",
-    "ember-fetch": "^5.1.1",
-    "ember-font-awesome": "^3.0.5",
-    "ember-load-initializers": "^1.0.0",
-    "ember-math-helpers": "^2.2.3",
+    "ember-cli-update": "^0.47.1",
+    "ember-component-css": "^0.6.9",
+    "ember-composable-helpers": "^3.0.2",
+    "ember-concurrency": "^1.1.3",
+    "ember-data": "~3.12.0",
+    "ember-export-application-global": "^2.0.1",
+    "ember-fetch": "^7.0.0",
+    "ember-font-awesome": "^3.1.1",
+    "ember-load-initializers": "^2.1.1",
+    "ember-math-helpers": "^2.7.0",
     "ember-maybe-import-regenerator": "^0.1.6",
-    "ember-mocha": "^0.12.0",
-    "ember-moment": "^7.4.1",
+    "ember-moment": "^7.6.0",
     "ember-native-dom-helpers": "^0.6.1",
-    "ember-notify": "^5.2.1",
+    "ember-notify": "^6.0.0",
     "ember-one-way-controls": "^3.0.1",
-    "ember-pad": "^1.2.2",
-    "ember-parachute": "^0.3.5",
-    "ember-power-calendar": "^0.7.3",
-    "ember-power-select": "^1.9.11",
-    "ember-resolver": "^5.0.1",
-    "ember-responsive": "^2.0.5",
-    "ember-route-action-helper": "^2.0.6",
-    "ember-scrollable": "^0.5.0",
-    "ember-simple-auth": "^1.4.0",
-    "ember-site-tour": "^1.1.0",
-    "ember-source": "~2.18.0",
-    "ember-test-selectors": "^1.0.0",
-    "ember-tether": "^1.0.0-beta.0",
-    "ember-truth-helpers": "^2.0.0",
-    "ember-validated-form": "^0.6.0",
-    "ember-wormhole": "^0.5.2",
+    "ember-pad": "^1.2.3",
+    "ember-parachute": "^1.0.2",
+    "ember-power-calendar": "^0.13.3",
+    "ember-power-calendar-moment": "^0.1.7",
+    "ember-power-select": "^2.3.4",
+    "ember-qunit": "^4.6.0",
+    "ember-resolver": "^7.0.0",
+    "ember-responsive": "^3.0.5",
+    "ember-route-action-helper": "^2.0.7",
+    "ember-scrollable": "^1.0.0",
+    "ember-simple-auth": "^2.1.1",
+    "ember-site-tour": "^2.2.0",
+    "ember-source": "~3.12.0",
+    "ember-test-selectors": "^3.0.0",
+    "ember-tether": "^1.0.0",
+    "ember-truth-helpers": "^2.1.0",
+    "ember-validated-form": "^1.4.2",
+    "ember-wormhole": "^0.5.5",
     "eslint-config-prettier": "^3.3.0",
-    "eslint-plugin-ember": "^5.0.1",
+    "eslint-plugin-ember": "^6.2.0",
+    "eslint-plugin-node": "^9.0.1",
     "eslint-plugin-prettier": "^2.1.2",
+    "faker": "^4.1.0",
     "intersection-observer": "^0.5.0",
-    "loader.js": "^4.1.0",
-    "pagination-pager": "^3.1.0",
-    "postcss-cssnext": "^3.0.2",
-    "postcss-import": "^12.0.0",
-    "postcss-responsive-type": "^1.0.0",
-    "prettier": "^1.7.4"
+    "loader.js": "^4.7.0",
+    "pagination-pager": "^3.2.0",
+    "prettier": "^1.7.4",
+    "qunit-dom": "^0.8.4",
+    "sass": "^1.24.2",
+    "typeface-source-sans-pro": "^0.0.75"
   },
   "greenkeeper": {
-    "ignore": ["ember-cli-htmlbars-inline-precompile", "ember-cli-mirage"]
+    "ignore": [
+      "ember-cli-mirage"
+    ]
```

Comment:
1. Why ignore `ember-cli-mirage`
2. Drop the usage of greenkeeper in favor of dependabot

Reply:
No idea :smile: 

I removed greenkeeper and configured Dependabot.